### DISCUSSION TITLE
공식 샘플 맵을 시험해보고 있는데 오류가 뜨네요 예제맵에서는 안그랬는데 이유가 뭔가요

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#795] [5. Q&A]
![제목 없음](https://user-images.githubusercontent.com/105163890/169979442-8110dac3-c2b9-4de2-9c29-65f3cf2dbca5.png)
![image](https://user-images.githubusercontent.com/105163890/169979538-7d742a20-5261-4f59-9b1f-10d86ccde732.png)
![image](https://user-images.githubusercontent.com/105163890/169979580-b7769d74-819f-4943-bfe2-c5cadee31044.png)
예제를 시험해보고 있는데 오류가 뜨네요 예제맵에서는 안그랬는데 이유가 뭔가요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://studio.zepeto.me/guides/script-import
위의 가이드를 참고하셔서 인스펙터에서 스크립트 연결이 누락된 것은 아닌지 확인 부탁 드립니다.

**Reply by PH62mem** [reply]
인스펙터에서 스크립트의 연결은 모두 확인했습니다.(말씀하시는 것이 유니티에서라면)
또다른 이유가 있을까요

**Reply by PH62mem** [reply]
방금 해결했습니다. 갑사합니다.
